test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -3580,7 +3580,7 @@ BOOST_AUTO_TEST_CASE(sha256_empty)
 	char const* sourceCode = R"(
 		contract C {
 			function f() returns (bytes32) {
-				return sha256();
+				return sha256("");
```

Comment:
Here the standard `R"(` works but on other cases you need a longer escape character `R"YY(` - why is that?

Reply:
This is weird, actually it should not work.